src/item.cpp

```diff
@@ -4621,16 +4623,58 @@ std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int t
                   contents_item.charges > 1 )
                 ? contents_item.charges
                 : 1;
+
+            // Color second half of container contents depending on perishability (highest perishability of all contents)
+            if( contents_item.is_food() ) {
+                // For comestibles
+                colorprefix = "<" + string_from_color( contents_item.color_in_inventory() ) + ">";
+                colorprefix.replace( 1, 1, "color" ); // changes <c_cyan> to <color_cyan>
+                if( contents.get_sealed_summary() == item_contents::sealed_summary::all_sealed ) {
+                    colorprefix = "<color_light_blue>";
+                }
+                colorsuffix = "</color>";
+            }
+
             contents_suffix_text = string_format( pgettext( "item name",
                                                   //~ [container item name] " > [inner item  name]"
-                                                  " > %1$s" ),
+                                                  " > %1$s%2$s" ),
+                                                  /* with_contents=false for nested items to prevent excessively long names */
+                                                  colorprefix, contents_item.tname( contents_count, true, 0, false )
                                                   /* with_contents=false for nested items to prevent excessively long names */
-                                                  contents_item.tname( contents_count, true, 0, false ) );
+                                                ) + string_format( "%s", colorsuffix );
+
         } else if( !contents.empty() ) {
-            contents_suffix_text = string_format( npgettext( "item name",
+
+            int worstrot = 0;
+            if( contents.get_sealed_summary() == item_contents::sealed_summary::all_sealed ) {
```

Comment:
Content of sealed container may still rot if the container has a spoil multiplier larger than 0. For example `bottle_plastic` has `"spoil_multiplier": 0.5`.

Reply:
Alright; can get the spoil multiplier in that case, will fix